app/flows/check_uk_visa_flow/outcomes/outcome_medical_n.erb

```diff
@@ -3,6 +3,9 @@
 <% end %>
 
 <% govspeak_for :body do %>
+
```

Comment:
do we need this extra line?

Reply:
No—it was split into two separate pieces of text (one for non-visa nationals, and one for EEA country nationals). See the next two added lines.